openpype/settings/handlers.py

```diff
@@ -235,6 +235,18 @@ def save_project_anatomy(self, project_name, anatomy_data):
         """
         pass
 
+    @abstractmethod
+    def save_change_log(self, project_name, changes, settings_type):
+        """Stores changes to settings to separate logging collection.
+
+        Args:
+            project_name(str, null): Project name for which overrides are
```

Comment:
Shouldn't then be `project_name=None` in the args?

Reply:
I copied docstring from different method, but `project_name` could actually be None, but adding this to signature would mean to add this to both `changes` and `settings_type` which are required. I wanted to keep `project_name` as first argument as it is in other methods.